modules/op-release-notes-1-14.adoc

```diff
@@ -0,0 +1,238 @@
+
+[id="tekton-chains-new-features-1-14_{context}"]
+=== {tekton-chains}
+
+* With this update, {tekton-chains} supports the `v1` value of the API version.
+
+* With this update, you can set the `artifacts.pipelinerun.enable-deep-inspection` parameter in the `TektonConfig` CR. When this parameter is `true`, {tekton-chains} records the results of the child task runs of a pipeline run. When this parameter is `false`, {tekton-chains} records the results of the pipeline run but not of its child task runs.
+
+* With this update, you can set the `builddefinition.buildtype` parameter in the `TektonConfig` CR to set the build type for in-toto attestation. When this parameter is `https://tekton.dev/chains/v2/slsa`, {tekton-chains} records in-toto attestations in strict conformance with the slsav1.0 specification. When this parameter is `https://tekton.dev/chains/v2/slsa-tekton`, {tekton-chains} records in-toto attestations with additional information such as the labels and annotations in each task run and pipeline run, and also adds each task in a pipeline run under `resolvedDependencies`.
+
+* Before this update, when {tekton-chains} was configured to use `gcs` storage, Tekton Chains did not record pipeline run information. With this update, {tekton-chains} records pipeline run information with this storage.
+
+* With this update, performance metrics are available for {tekton-chains}. To access the metrics, expose the `tekton-chains-metrics` service and then usr the /metrics path on this service, port 9090. These metrics are also available in the {OCP} Monitoring stack.
+
+* With this update, {tekton-chains} uses the new `v2alpha3` record format version when recording pipeline runs and task runs that use the `v1` version value.
+
+* With this update, {tekton-chains} uses the `v1` version of pipeline run and taskline run formats internally.
+
+[id="tekton-results-new-features-1-14_{context}"]
+=== {tekton-results}
+
+* With this update, if {tekton-results} is installed, {tekton-results} records the summary and record data for pipeline runs started using {pac}.
+
+* With this update, {tekton-results} provides up to 100 megabytes of logging information for a pipeline or task.
+
+* With this update, any authenticated user can view the `tekton-results-api-service` route in the openshift-pipelines namespace to interact with Tekton Results using a REST API.
+
+* With this update, the {tekton-results} API includes a new endpoint for fetching summary and aggregation for a list of records.
+
+* With this update, the `GetLog` endpoint of the {tekton-results} API returns raw bytes with the `text/plain` content type.
+
+With this update, you can optionally specify a custom CA certificate in the `options.Data.config.tekton-results-api-config.DB_SSLROOTCERT` spec in the `TektonResult` CR. In this value. In this case, Tekton Results requires an SSL connection to the database server and uses this certificate for the connection.
```

Comment:
If we want to use the options field in the TektonResult spec, then we need to use it like this:
```
  options:
    configMaps:
      tekton-results-api-config:
        data:
          config: |
            DB_HOST=tekton-results-postgres-service.openshift-pipelines.svc.cluster.local
            DB_PORT=5432
            DB_NAME=tekton-results
            DB_SSLMODE=disable
            DB_SSLROOTCERT=xyz
            DB_ENABLE_AUTO_MIGRATION=true
            SERVER_PORT=8080
            PROMETHEUS_PORT=9090
            PROMETHEUS_HISTOGRAM=false
            TLS_PATH=/etc/tls
            AUTH_DISABLE=true
            AUTH_IMPERSONATE=false
            LOG_LEVEL=debug
            LOGS_API=true
            LOGS_TYPE=File
            LOGS_BUFFER_SIZE=32768
            LOGS_PATH=/logs
```

Reply:
@khrm does this mean that all the other DB settings must also, in this case, be provided in this place and not where they normally are directly in the Tekton Result spec? Or will this work:

```
  apiVersion: operator.tekton.dev/v1alpha1
  kind: TektonResult
  metadata:
    name: result
  spec:
    targetNamespace: openshift-pipelines
    logs_api: true
    log_level: debug
    db_port: 5432
    db_host: tekton-results-postgres-service.openshift-pipelines.svc.cluster.local
    logs_path: /logs
    logs_type: File
    logs_buffer_size: 32768
    auth_disable: true
    tls_hostname_override: tekton-results-api-service.openshift-pipelines.svc.cluster.local
    db_enable_auto_migration: true
    server_port: 8080
    prometheus_port: 9090
    options:
      configMaps:
        tekton-results-api-config:
          data:
            config:
               DB_SSLROOTCERT: <certificate>
```

Reply:
All settings should be provided in options only.

Reply:
@khrm well, I don't know which settings are in that configMap. Those I document are listed in my example, Are any of these settings in the tekton-results-api-config  configMap? Such as server_port, prometheus_port, db_port, db_host ?

Reply:
These are the settings:
```
DB_HOST=tekton-results-postgres-service.tekton-pipelines.svc.cluster.local DB_PORT=5432 DB_NAME=tekton-results DB_SSLMODE=disable DB_SSLROOTCERT= DB_ENABLE_AUTO_MIGRATION=true SERVER_PORT=8080 PROMETHEUS_PORT=9090 PROMETHEUS_HISTOGRAM=false TLS_PATH=/etc/tls AUTH_DISABLE=false AUTH_IMPERSONATE=true LOG_LEVEL=info LOGS_API=false LOGS_TYPE=File LOGS_BUFFER_SIZE=32768 LOGS_PATH=/logs S3_BUCKET_NAME= S3_ENDPOINT= S3_HOSTNAME_IMMUTABLE=false S3_REGION= S3_ACCESS_KEY_ID= S3_SECRET_ACCESS_KEY= S3_MULTI_PART_SIZE=5242880 GCS_BUCKET_NAME= STORAGE_EMULATOR_HOST=
```